tests/unit/transports/socket_transport_test.py

```diff
@@ -25,51 +25,79 @@ def test_init(self):
         assert_equals(bytearray(), self.transport._buffer)
         assert_true(self.transport._synchronous)
 
-    def test_connect_with_no_klass_arg_ipv4(self):
-        sock = mock()
-        expect(socket.socket).returns(sock)
-
+    def _set_up_connect_test(self, sock):
+        """Set up common options and expects for connect() related tests."""
         self.connection._connect_timeout = 4.12
         self.connection._sock_opts = {
             ('family', 'tcp'): 34,
             ('range', 'ipv6'): 'hex'
         }
-
         expect(sock.setblocking).args(True)
         expect(sock.settimeout).args(4.12)
         expect(sock.setsockopt).any_order().args(
             'family', 'tcp', 34).any_order()
         expect(sock.setsockopt).any_order().args(
             'range', 'ipv6', 'hex').any_order()
-        expect(socket,'getaddrinfo').args('host',5309,0,0,socket.IPPROTO_TCP).returns(
-            [('family','socktype','proto', 'canon',('host.net',5309))] )
-        expect(sock.connect).args(('host.net', 5309))
         expect(sock.settimeout).args(None)
 
+    def test_connect_with_no_klass_arg_ipv4(self):
+        sock = mock()
+        expect(socket.socket).returns(sock)
+        self._set_up_connect_test(sock)
+        expect(socket, 'getaddrinfo').args(
```

Comment:
Any reason why this is `expect(socket, 'getaddrinfo')` vs. `expect(socket.getaddrinfo)`. I'm new to Chai, so just curious?